python/docs/models/OpenConsumerCreateRequestDTO.md

```
@@ -8,15 +8,15 @@ dict, frozendict.frozendict,  | frozendict.frozendict,  |  |
 ### Dictionary Keys
 Key | Input Type | Accessed Type | Description | Notes
 ------------ | ------------- | ------------- | ------------- | -------------
-**appId** | str,  | str,  | 第三方应用ID | [optional]
+**ownerName** | str,  | str,  | 负责人用户名 |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)